test/samples/sub_sdh.srt

```diff
@@ -0,0 +1,3 @@
+1
+00:00:00,000 --> 00:00:02,000
+MAN: Filter (SOME) (things) from [this] （random） string.
```

Comment:
If we are testing everything , we could add lines that should be completely removed. Also lines with "mismatched" parentheses to make sure those are not filtered.